akka-operator/values.yaml

```diff
@@ -38,3 +38,22 @@ resources:
 nodeSelector: {}
 tolerations: []
 affinity: {}
+
+# valid values:
+# - none (default)
```

Comment:
I guess a more standard default will be `false` or `nil`:
https://helm.sh/docs/chart_template_guide/control_structures/#ifelse

Reply:
done ..